lib/screens/home/widgets/category_main_menu.dart

```diff
@@ -19,24 +21,35 @@ class _CategoryMainMenuState extends ConsumerState<CategoryMainMenu> {
     final gridTile = ref
         .read(homeViewModelProvider.notifier)
         .cocktailMenuTiles
-        .map((tile) => _buildGridTile(withName: tile.name, path: tile.path))
+        .map(
+          (tile) => _buildGridTile(
+              withName: tile.name, path: tile.path, image: tile.image),
```

Comment:
might less verbose to just to pass the complete object if we basically need it all.
```suggestion
              withTile: tile),
```